calendars/data-umbrella.yaml

```diff
@@ -1,15 +1,24 @@
 name: Data Umbrella Events Calendar
 events:
+  - summary: Make Your Own JupyterLab Extension
+    description: |
+      Make your own JupyterLab extension in this tutorial. JupyterLab is a free and open source, browser based integrated development environment (IDE) that is the successor to Jupyter Notebook. Designed from the core to be extensible, JupyterLab is itself an extension of the Jupyter framework. Since its release, open source developers have contributed hundreds of third party extensions (652 third-party extensions as of 2024-10-15).
+      Meetup: https://www.meetup.com/data-umbrella/events/304917194/
+      Webinarlink: https://www.bigmarker.com/neo4j/Data-Umbrella-Webinar
+      Video: https://www.youtube.com/@DataUmbrella
+    begin: 2025-01-07 17:00:00 +00:00
+    end: 2025-01-07 18:00:00 +00:00
+    url: https://www.bigmarker.com/neo4j/Data-Umbrella-Webinar
+
   - summary: Introduction to Git
     description: |
       Join us for an introductory session on Git. Discover some
       fundamental concepts, explore its practical applications, and
       learn how to efficiently use both the command line and the Visual
       Studio Code graphical user interface (GUI).
-
```

Comment:
Quick check: were these separator lines meant to be removed? There's one below as well.

Reply:
Woops - Fixed that

Reply:
Empty lines needed, I think!

Reply:
Oh - thought you were saying we needed a `-` to make the line break. I think I’ve fixed it?